src/password.rs

```diff
@@ -68,15 +69,9 @@ impl SafePassword {
 }
 
 /// An error for parsing a password from a string
-#[derive(Debug)]
+#[derive(Debug, Snafu)]
 pub struct PasswordError;
```

Comment:
I would prefer this to be an enum with one variant and a better description. Where is this used?

Reply:
It is intrinsically used within the `WalletConfig` and `Cli` structs

Reply:
`PasswordError` removed, thanks for suggesting use of `Infallible` instead.